scripts/benchmarks/load-limit-cloud/README.md

```
@@ -0,0 +1,385 @@
+|---|---|---|---|
+| Fargate 0.5 vCPU | `csarena-backend-bench-512` | 512 / 1024 | `task-definitions/bench-512.json` |
+| Fargate 1 vCPU | `csarena-backend-bench-1024` | 1024 / 2048 | `task-definitions/bench-1024.json` |
+| Fargate 2 vCPU | `csarena-backend-bench-2048` | 2048 / 4096 | `task-definitions/bench-2048.json` |
+
+각 환경 × 룸 수 단계(50, 100, 200, 500) = **총 12회 측정.**
+
+## 시뮬레이션 시나리오
+
+12번과 동일 (`scripts/benchmarks/load-limit/load-test-rooms.js` 그대로 재사용). `NGINX_URL` 환경변수로 endpoint만 교체.
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)